person.rb

```diff
@@ -1,8 +1,10 @@
+require './nameable'
```

Comment:
- [ ] You are including the nameable class but not extending it, the Person class should extend nameable class

Reply:
We are creating `def correct_name`. Doesn't it need `nameable` import?